doc/content/getting_started/hpc.md

```diff
@@ -47,9 +47,54 @@ module load use.moose moose-dev
 
 <!-- Re-use the bulk of the build and test section from the main getting started instructions here -->
 
-!include getting_started/installation.md start=Then navigate to the TMAP8 end=TMAP8 can be compiled and tested include-end=True
+Then navigate to the TMAP8 clone directory and download the MOOSE submodule:
 
 ```bash
+cd ~/projects/TMAP8
+git submodule update --init moose
+```
+
+!alert! tip title=Thermochimica library (optional)
+To have access to the optional thermochemistry library Thermochimica [!cite](piro2013) provided within
+the MOOSE [modules/chemical_reactions/index.md], check out the corresponding submodule by performing
+the following before building:
+
+```bash
+cd ~/projects/TMAP8/moose
+git submodule update --init --checkout modules/chemical_reactions/contrib/thermochimica
+```
+!alert-end!
+
+!alert note
+The copy of MOOSE provided with TMAP8 has been fully tested against the current
+TMAP8 version, and is guaranteed to work with all current TMAP8 tests.
+
+<!-- End getting started re-use -->
+
+Once MOOSE is downloaded, several MOOSE dependencies need to be built: PETSc, libMesh, and WASP.
+These can be built easily using update-and-rebuild scripts within MOOSE. To speed up these build
+times, it is highly recommended to set the environment variable `MOOSE_JOBS`, which will speed up
+compilation time by using more than one core to build the dependency. One can set this variable to
+signal that the scripts use of 8 computer cores by doing the following:
+
+```bash
+export MOOSE_JOBS=8
```

Comment:
```suggestion
export MOOSE_JOBS=4 METHODS=opt 
```
Didn't @loganharbour send a message on slack about this needing to be 4 now? 
I'm also adding METHODS to build opt.

Reply:
Broadcasted after I made this 😄

Reply:
Hahaha I figured 😄 .